Automattic/wp-calypso
# Remove trail map tracks traces

Follow up to https://github.com/Automattic/wp-calypso/pull/93706

## Proposed Changes

This removes the blocking loading of trail map experiment and removes tracks property with it.

It also disabled the guided onboarding flow in a better fashion.

## Why are these changes being made?
To clean up tracks and avoid tech debt.

## Testing Instructions

Please test thoroughly.

1. Go through the onboarding flow by visiting /start.
2. Make a site with a free domain and a free plan.
3. Make a site with a free domain and a paid plan.
4. When picking a paid domain, the free plan should cease being a column, and should be in the subheader.
5. When clicking the CTA in the subheader, you should see a modal "A paid plan is required for your domain."
6. Please try all three options in the modal by going through the flow 3 times.
7. Pick a paid domain and a paid plan. You should land in checkout.
